src/features/auth/services/logout.ts

```diff
@@ -0,0 +1,6 @@
+import { api } from '@/features/core';
+
+export const logout = async () => {
+  const res = await api.DELETE('/auth/logout');
+  if (res.error) throw res.error;
+};
```

Comment:
#42 처럼 `src/data/delete-auth-logout.ts`로 위치를 바꾸고, 에러를 throw하지 말고 type-safe한 에러를 리턴하게 했으면 좋겠어요

```suggestion
import type { ErrorStatus } from 'openapi-typescript-helpers';

import { api } from '@/features/core';

export const deleteAuthLogout = async () => {
  const { data, error, response } = await api.DELETE('/auth/logout');
  
  if (error || !data) {
      const status = response.status as Extract<
        keyof paths['/auth/logout']['delete']['responses'],
        ErrorStatus
      >;
      
      return { error, status };
  }
  
  return { data };
};
```

Reply:
확인했습니당~!